common/thorhelper/CMakeLists.txt

```diff
@@ -68,6 +68,8 @@ set (    SRCS
          thorrparse.ipp
          thorsort.cpp
          thorsort.hpp
+         ${HPCC_SOURCE_DIR}/esp/platform/persistent.cpp
```

Comment:
Feels wrong for thorhelper to be including files from esp/platform.

Is there a better structure that could be considered? For example, should persistent.cpp/hpp be in thorhelper dll (and exported from there rather than copied into other places)?

Reply:
thorhelper does seem to be a more common component, let me give it a try...

Reply:
Would it make sense to put them under jlib instead? The function seems to be generic enough, also would help avoid adding thorhelper to the include path of quite some projects. @richardkchapman

Reply:
@mayx Depends what they do, but quite possibly...

Reply:
I tried this option and it does make the change much easier. As it only depends on jlib so it doesn't bring other changes to jlib other than the 2 files themselves. Before I check in my changes, I just want to make sure that @mckellyln and @jakesmith are OK with it?

Reply:
Not sure about moving it to jlib, it doesn't quite feel core enough or likely to be generically used elsewhere to me.
thorhelper (despite its misleading name) is a 2nd tier support library used by quite a few components, including esp, and this feels like it would be more at home there.
@mayx

Reply:
OK I'll move it to thorhelper.

Reply:
@jakesmith I moved the files to thorhelper and added thorhelper to the include path of the projects that need it. There's no change to the code itself. Please have a quick review when you have a chance...